randovania/games/samus_returns/gui/preset_settings/msr_patches_tab.py

```diff
@@ -41,6 +41,14 @@ def tab_title(cls) -> str:
     def uses_patches_tab(cls) -> bool:
         return True
 
+    def _add_persist_option(self, check: QtWidgets.QCheckBox, attribute_name: str):
```

Comment:
Mypy errors could be fixed or just remove the whole thing if you don't need it. (See my previous comment)